cmd/treedb_text_hybrid_scale/main.go

```
@@ -0,0 +1,1624 @@
+		}
+		b = strconv.AppendFloat(b, float64(value), 'f', -1, 32)
+	}
+	b = append(b, `]}`...)
+	return b
+}
+
+func appendJSONStringContent(dst []byte, s string) []byte {
+	// Inputs are controlled fixture tokens without quotes/backslashes. Keep this
+	// tiny helper so future shape edits do not accidentally write raw JSON quotes.
+	if strings.ContainsAny(s, "\\\"") {
+		quoted, _ := json.Marshal(s)
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked

(errchkjson)